slack-gamebot/commands/taunt.rb

```diff
@@ -0,0 +1,20 @@
+module SlackGamebot
+  module Commands
+    class Taunt < SlackRubyBot::Commands::Base
+      def self.call(client, data, match)
+        taunter = ::User.find_create_or_update_by_slack_id!(client, data.user)
+        if taunter.registered?
+          arguments = match['expression'].split.reject(&:blank?) if match['expression']
+          arguments ||= []
+          taunt = ::Taunt.create_from_teammates_and_opponents!(client.owner, data.channel, taunter, arguments)
```

Comment:
What's this `Taunt` class? It doesn't seem to be included in the PR, but the spec passes because it's hiding the error.